examples/petsc/bpsdmplex.c

```diff
@@ -100,6 +100,45 @@ static int CreateRestrictionPlex(Ceed ceed, CeedInt P, CeedInt ncomp,
   PetscFunctionReturn(0);
 }
 
+// BC functions
+PetscErrorCode BCsMass(PetscInt dim, PetscReal time, const PetscReal x[],
+                       PetscInt vscale, PetscScalar *u, void *ctx) {
+  for (PetscInt i = 0; i < vscale; i++)
+    u[i] = PetscSqrtScalar(PetscSqr(x[0]) + PetscSqr(x[1]) +
+                           PetscSqr(x[2]));
+
+  return 0;
+}
```

Comment:
This is never called due to `enforce_bcs = false`, so why not just use `NULL`?

Reply:
Currently `enforce_bcs` is a command line option, so a user could call `-enforce_bcs -problem bp1`.